main.py

```diff
@@ -169,22 +170,20 @@ def subModeToggle():
         print(subMode)
     chat(s, "Sub Goal Madness has been " + subStatusMessage)
 
-def timeCheck(secondDuration): #checks time passed and sends a message
-    global timerCommandClock
+def resetWasd(secondDuration): #checks time passed and sends a message
```

Comment:
The code and comments imply there might be future timer-based stuff, in which case this should:
- Rename this to something like `resetTimer`
- Pass a 'reset' or 'stop' function reference
- When a timer-based function is started, `resetTimer` is called with the duration and stop function as args within a thread (or this function can be refactored such that it does all the threading for you so callers don't have to worry about that).